Samples~/WebGLExample/Editor/EditorWindows/ExampleSetup.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using ReadyPlayerMe.Core;
+using ReadyPlayerMe.Samples;
+using UnityEditor;
+using UnityEngine;
+
+namespace ReadyPlayerMe.Samples
+{
+    public static class ExampleSetup
+    {
+        private const string TAG = nameof(ExampleSetup);
+        private static string WINDOW_TITLE = "RPM WebGL Example";
+        private static string DESCRIPTION = "This example includes a WebGL template that can be used for WebGL builds. To use the template it needs to be moved inside WebGLTemplates folder and set in the player settings. Would you like to move it automatically?";
+        private const string CONFIRM_BUTTON_TEXT = "Ok";
+        private const string CANCEL_BUTTON_TEXT = "Cancel";
+        
+        private static readonly string RPM_WEBGL_SCREEN_SHOWN_KEY = "rpm-webgl-screen-shown";
+        
+        [InitializeOnLoadMethod]
+        private static void InitializeOnLoad()
+        {
+            if (ProjectPrefs.GetBool(RPM_WEBGL_SCREEN_SHOWN_KEY))
+            {
+                return;
+            }
+
+            ProjectPrefs.SetBool(RPM_WEBGL_SCREEN_SHOWN_KEY, true);
+            EditorApplication.update += ShowWebGLScreen;
+        }
+
+        private static void ShowWebGLScreen()
+        {
+            var shouldUpdate = EditorUtility.DisplayDialogComplex(WINDOW_TITLE,
+                DESCRIPTION,
+                CONFIRM_BUTTON_TEXT,
+                CANCEL_BUTTON_TEXT,
+                "");
+
+            switch (shouldUpdate)
+            {
+                // Update
+                case 0:
+                    OnConfirm();
+                    break;
+            }
+            EditorApplication.update -= ShowWebGLScreen;
+        }
+        
+        private static void OnConfirm()
+        {
+            var templatePaths = GetTemplatePaths();
+            
+            if (templatePaths == null)
+            {
+                Debug.LogWarning("Failed to set source and destination paths. No changes were done to project");
+                return;
+            }
+            Copy(templatePaths[0], templatePaths[1]);
+            SetWebGLTemplate();
+        }
+        private static List<string> GetTemplatePaths()
+        {
+            string[] res = Directory.GetFiles(Application.dataPath, "ExampleSetup.cs", SearchOption.AllDirectories);
+            if (res.Length == 0)
+            {
+                return null;
+            }
+            string path = res[0].Replace("ExampleSetup.cs", "").Replace("\\", "/");
+            var sourcePath = path.Substring(0, path.IndexOf("/Editor/")) + "/WebGLTemplates/RPMTemplate";
```

Comment:
Small improvement suggestion. 
Some of the same strings are used in multiple places, they should be moved into constants to improve maintainability.